src/steps/generic.rs

```diff
@@ -433,11 +433,25 @@ pub fn run_vscodium_extensions_update(ctx: &ExecutionContext) -> Result<()> {
         .lines()
         .next()
     {
-        Some(item) => Version::parse(item).map_err(std::convert::Into::into),
+        Some(item) => {
+            // Strip leading zeroes because `semver` does not allow them, but VSCodium uses them sometimes
+            let item = item
+                .split('.')
+                .map(|s| s.trim_start_matches('0'))
+                .collect::<Vec<_>>()
+                .join(".");
+            Version::parse(&item).map_err(std::convert::Into::into)
+        }
         _ => return Err(SkipStep(String::from("Cannot find vscodium version")).into()),
     };
 
-    if !matches!(version, Ok(version) if version >= Version::new(1, 86, 0)) {
+    // Raise any errors in parsing the version
+    //  The benefit of handling VSCodium versions so old that the version format is something
+    //  unexpected is outweighed by the benefit of failing fast on new breaking versions
+    let version = version.wrap_err("the output of `codium --version` changed, please file an issue to Topgrade")?;
```

Comment:
> When the version is incorrectly parsed (is Err(_)), instead of assuming the version is too old, throw it as an error. 

I think we should port this to the VSCode step as well

Reply:
There are only 2 steps, so it is ok to allow a tiny amount of duplications, IMHO

Reply:
IMO the fact that I forgot to apply it to the other as well proves that its better to deduplicate it. Especially now that it gets more complex with the zero stripping. But if you want I'll keep it this way.

Reply:
You can try it in a commit, and if we are not overengineering:

1. Allow incompatible changes between VSCode and VSCodium
2. The whole thing is still easy to understand and maintain

I will accept it